client/app/views/templates/popover_event.jade

```diff
@@ -1,31 +1,36 @@
-.line
-    if allDay
-        span.timeseparator=t('all day during')
-        span &nbsp;
-        input#input-diff.col-xs2.input-mini(type="number", value=model.getDiff(),''
-        placeholder=1, min=1)
-        span &nbsp;
-        span.timeseparator= ' ' + t('days')
+.popover-content-wrapper
+    label.timed(aria-hidden="#{allDay}")
+        span.caption= t("from")
+        input.input-start.input-time(tabindex="4",
+                                     type="time",
+                                     size="5"
+                                     placeholder=t("From hours:minutes"),
+                                     value=start.format(tFormat))
 
-    else
-        span.timeseparator=t("from")
-        input#input-start.focused.input-mini(type="time", placeholder=t("From hours:minutes"), value=model.getStartDateObject().format(dtFormat))
-        span &nbsp;
-        span.timeseparator=t("to")
-        input#input-end.input-mini(type="time", placeholder=t("To hours:minutes+days"), value=model.getEndDateObject().format(dtFormat))
-        span &nbsp;
-        input#input-diff.col-xs2.input-mini(type="number", value=model.getDiff(),''
-        placeholder=0, min=0)
-        span &nbsp;
-        span.timeseparator= ' ' + t('days later')
+    label.aside
+        input.input-allday(tabindex="3",
+                           type="checkbox",
+                           value='checked',
+                           checked=allDay)
+        span.caption= t('All day')
 
-.line
-    input#input-desc.input(type="text", value=model.get("description"),
-        placeholder=t("summary"))
-    input#input-place.input-small(type="text", value=model.get("place"),
-        placeholder=t("Place"))
+    label
+        span.caption= t("To")
```

Comment:
My Bad™… I primarily use the _to_ string then forget to change it afterwards :disappointed:. Thanks.